### DISCUSSION TITLE
How to run/use

### DISCUSSION THREAD

**Discussion by Rubiksmath** [preda/gpuowl#233] [Q&A]
hi, was wanting to know how to run this, seems cool. the instructions provided on how to run didnt seem very clear to me, i am aware is says i require a few things for it to work, but i am not 100% sure what to do with them once i have them. i might have a slight idea of what to do, but as an idiot its not entirely clear what i do (and i cant find a tutorial anywhere). could someone elaborate on how to go about starting this thing up? i am running on a windows 10 os with ryzen 5 5600x and an rtx 3060. ( I also wanted to use the regular prime95 application found on gimps but the download link does absolutely nothing.)

**Comment by valeriob01** [comment]
hi, you should ask on the forum, mersenneforum.org , there are threads describing specific usage for Windows, along with gpuOwl executables for Windows. Look at this link https://mersenneforum.org/showthread.php?t=23391 which is a thread from a user named "kriesel" who posts gpuOwl reference material specific for Windows.

**Reply by Rubiksmath** [reply]
okay, sure ill try, ive been doing stuff for the last 7 hours trying to get it to work, it turns out it might be a linux only thing? also the c++ compiler was saying the makefile is omitting a command or something. ill try that thanks

**Reply by Rubiksmath** [reply]
wow this is like weird the download is a 7z file that i need 7 zip to extract, as an idiot i never unsderstand why programs are packaged like that, instead of just a regular folder. oh wow how har dcan this be i new need to know like everything about my computer and make a config.txt manually? oh man

**Reply by Rubiksmath** [reply]
Okay, ive sort of got something working, i dont have a config.txt since i have no clue what the hell to put there, but it runs for a few seconds and then encounters 5 different compilation/open cl specification errors, where would be best to post the error messages and try and find a solution? i dont know why these things have to be so damn hard, especialy considering the fact i would call myself very proficient with computer use. i think im going to give up since its becoming far too hard and it shouldnt take 10 hours to get something to work, im clearly just too dumb.

**Reply by valeriob01** [reply]
Computing primes requires patience, and to read the documentation in the forum and github. The config.txt file accepts the same options that you see by running `gpuowl -h`
The opencl errors possibly because you need some extra libraries or dll.

**Reply by valeriob01** [reply]
also for NVIDIA you will need to use the `-yield` option

**Reply by Rubiksmath** [reply]
okay, i got it to work, although its not particularly useful since its twice as slow as the cpu (ms/iter 10 vs 5), but i guess i can run 2 tests at once now without sacrificing much cpu

**Reply by valeriob01** [reply]
What GPU do you have ?

**Reply by valeriob01** [reply]
> What GPU do you have ?

Sorry , rtx 3060, ok, I can't say for it, but Radeon VII is faster than cpu.
